docs/products/kafka/kafka-connect/howto/mongodb-sink-mongo.rst

```diff
@@ -0,0 +1,149 @@
+
+    This document refers to the MongoDB sink connector by MongoDB, you can browse the Lenses.io implementation in the :doc:`related document <mongodb-sink-lenses>`
+
+.. _connect_mongodb_sink_prereq:
+
+Prerequisites
+-------------
+
+To setup a MongoDB sink connector, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`.
+
+Furthermore you need to collect the following information about the target MongoDB database upfront:
+
+* ``MONGODB_CONNECTION_URI``: The MongoDB database connection URL in the format ``mongodb://USERNAME:PASSWORD@HOST:PORT`` where:
+    
+    * ``USERNAME``: The database username to connect
+    * ``PASSWORD``: The password for the username selected
+    * ``HOST``: the MongoDB hostname
+    * ``PORT``: the MongoDB port
+
+* ``MONGODB_DATABASE_NAME``: The name of the MongoDB database
+* ``TOPIC_LIST``: The list of topics to sink divided by comma
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PORT``: The Apache Kafka's schema registry port, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_USER``: The Apache Kafka's schema registry username, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PASSWORD``: The Apache Kafka's schema registry user password, only needed when using Avro as data format
+
+
+.. Note::
+
+    The Apache Kafka related details are available in the `Aiven console <https://console.aiven.io/>`_ service *Overview tab* or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+    The ``SCHEMA_REGISTRY`` related parameters are available in the Aiven for Apache Kafka® service page, *Overview* tab, and *Schema Registry* subtab
+
+    As of version 3.0, Aiven for Apache Kafka no longer supports Confluent Schema Registry. For more information, read `the article describing the replacement, Karapace <https://help.aiven.io/en/articles/5651983>`_
+
+Setup a MongoDB sink connector with Aiven Console
+----------------------------------------------------
+
+The following example demonstrates how to setup a MongoDB sink connector for Apache Kafka using the `Aiven Console <https://console.aiven.io/>`_.
+
+Define a Kafka Connect configuration file
+'''''''''''''''''''''''''''''''''''''''''
+
+Define the connector configurations in a file (we'll refer to it with the name ``mongodb_sink.json``) with the following content, creating a file is not strictly necessary but allows to have all the information in one place before copy/pasting them in the `Aiven Console <https://console.aiven.io/>`_:
+
+.. code-block:: json
+
+    {
+        "name":"CONNECTOR_NAME",
+        "connector.class": "com.mongodb.kafka.connect.MongoSinkConnector",
+        "topics": "TOPIC_LIST",
+        "connection.uri": "MONGODB_CONNECTION_URI",
```

Comment:
Similar point on the connection.uri as in the other article! the definition below is way more helpful :)